### ISSUE TITLE
Warning on cargo_common.lua

### ISSUE THREAD

**Issue by Mutos** [naev/naev#1490]
Hello all,

I got a recurring warning on cargo_common.lua whenever I run NAEV (commit 5ce7b3707, tested on Ubuntu 20.04, not tested yet on Windows 7/10).

Trace is the following : 

```
Warning: [misn_runFunc] Mission 'Cargo Rush' -> 'create': [string "cargo_common.lua"]:201: bad argument #1 to 'format' (string expected, got userdata)
stack traceback:
	[C]: in function 'format'
	[string "cargo_common.lua"]:201: in function 'cargo_setDesc'
	[string "missions/neutral/cargo_rush.lua"]:127: in function <[string "missions/neutral/cargo_rush.lua"]:77>
```

I think it might be quite simple, but I fear not to have time to search and solve it. I may try but no guarantee of success...

@+

Benoît 'Mutos' Robin

**Comment by Mutos** [comment]
Proposed fix in PR #1491

**Comment by bobbens** [comment]
This seems very critical. Only affects Lua and not LuaJIT apparently though. Might warrant another beta...
